client/my-sites/theme/main.jsx

```diff
@@ -335,23 +335,23 @@ const ThemeSheet = React.createClass( {
 					sourcePath={ `/theme/${ this.props.id }${ section ? '/' + section : '' }` }
 				/> }
 				{ this.state.showPreview && this.renderPreview() }
-				<HeaderCake className="themes__sheet-action-bar"
+				<HeaderCake className="theme__sheet-action-bar"
 							backHref={ this.props.backPath }
 							backText={ i18n.translate( 'All Themes' ) }>
-					<Button className="themes__sheet-primary-button" onClick={ this.onPrimaryClick }>
+					<Button className="theme__sheet-primary-button" onClick={ this.onPrimaryClick }>
 						{ actionTitle }
 						{ ! this.isActive() && priceElement }
 					</Button>
 				</HeaderCake>
-				<div className="themes__sheet-columns">
-					<div className="themes__sheet-column-left">
-						<div className="themes__sheet-content">
+				<div className="theme__sheet-columns">
+					<div className="theme__sheet-column-left">
+						<div className="theme__sheet-content">
 							{ this.renderSectionNav( section ) }
 							{ this.renderSectionContent( section ) }
-							<div className="footer__line"><Gridicon icon="my-sites" /></div>
+							<div className="theme__footer_line"><Gridicon icon="my-sites" /></div>
```

Comment:
`theme-page` could also work. Singular section names have the issue that they clash with components and we need to clarify it. Once we move application components into a specific folder we may have a better ground for this.